### ISSUE TITLE
Color escape sequences in output

### ISSUE THREAD

**Issue by exploide** [diego-treitos/linux-smart-enumeration#32]
Since the script was made compatible with `/bin/sh` instead of relying on bash (which I think was a good step), I noticed some weird behaviour on some targets.

Specifically, the color escape sequences are not interpreted but remain in the output when `/bin/sh` is a not so recent version of dash.
Even more disturbing, passing `-c` to lse to suppress colors completely does not have any effect here.
Running explicitly with bash is no problem.

The issue occurs for example on Ubuntu 16.04 with dash 0.5.8-2.1ubuntu2 (and older).

The issue does no longer occur since Ubuntu 18.04 with dash 0.5.8-2.10.

This can most easily be verified in a Docker/Podman container:

```
podman run --rm -it ubuntu:16.04
apt update && apt install wget
wget "https://github.com/diego-treitos/linux-smart-enumeration/raw/master/lse.sh" -O lse.sh
chmod 700 lse.sh
./lse.sh
```

**Comment by diego-treitos** [comment]
@Exploide Thanks a lot for keeping reporint bugs.

This is due to dash using the builtin printf instead of the command. It should be solved in version 2.3.

**Comment by oabsixr** [comment]
Having a similar issue with the color running lse without explicitly running with bash on an older device. A screenshot with some details, not sure if it's enough information for you?

![image](https://user-images.githubusercontent.com/67179369/91378783-de6f7f00-e853-11ea-85f4-8efccd039eb8.png)

![image](https://user-images.githubusercontent.com/67179369/91378709-af590d80-e853-11ea-9f36-e7bf78d399bf.png)
